wavefront_sdk/common/utils.py

```diff
@@ -371,6 +373,32 @@ def event_to_json(name, start_time, end_time, source, tags,
     return str(json.dumps(event))
 
 
+def get_sem_ver(name):
+    """Returns semantic version of sdk used in Wavefront reportable format.
+    Ex: <major>.<2-digit-minor><2-digit-patch> (1.0603 => v1.6.3)
+
+    @param name: SDK Name
+    @type name: str
+    @return: Semantic version in wavefront format as String
+    """
+    version = pkg_resources.require(name)[0].version
```

Comment:
`pkg_resources.require(name)` should we have try-catch for `DistributionNotFound` exception if the given name is invalid?